packages/sdk/wallets/wallet-walletconnect/src/wc-signer.ts

```diff
@@ -0,0 +1,68 @@
+import { CeloTx, EncodedTransaction, Signer } from '@celo/connect'
+import { EIP712TypedData } from '@celo/utils/src/sign-typed-data-utils'
+import WalletConnect from '@walletconnect/client'
+import { SessionTypes } from '@walletconnect/types'
+import * as ethUtil from 'ethereumjs-util'
```

Comment:
Should this be listed in `dependencies` or does it come from somewhere else?

Reply:
Yes it should